src/molecules/formfields/GoogleAutoCompleteField/index.js

```diff
@@ -51,6 +51,21 @@ class GoogleAutoCompleteField extends Component {
   updateAddress = () => {
     const place = this.state.autocomplete.getPlace();
     const addressData = this.formatAddressInfo(place.address_components);
+
+    if (!addressData.number) {
+      const { value } = this.props.input;
+
+      if (value) {
+        const matches = value.match(/^(\d+)/);
+
+        if (matches && matches[1]) {
+          const number = parseInt(matches[1], 10);
```

Comment:
Is there a preference for returning something at index 1 instead of index 0? Also, is there an advantage of using `parseInt` over `Number()`?

Reply:
I actually never use `Number()` because I prefer not to instantiate classes of primitives using constructors; I've actually always just used `parseInt`/`parseFloat` in situations like this.

As for the first question, there actually isn't! Good catch, I'll change it - I had done `[1]` just out of habit to capture the first group, but since the first group is all that we're matching anyway, it's pretty redundant.